src/EditorFeatures/TestUtilities/Options/OptionSerializerTests.cs

```diff
@@ -88,10 +88,7 @@ public void SerializationAndDeserializationForEnum()
     [Fact]
     public void SerializationAndDeserializationForNullableEnum()
     {
-        var options = new IOption2[]
-        {
-            SolutionCrawlerOptionsStorage.SolutionBackgroundAnalysisScopeOption,
-        };
+        var options = Array.Empty<IOption2>();
```

Comment:
>var options = Array.Empty<IOption2>(); [](http://example.com/codeflow?start=8&length=38)

seems like the loop below might be done very quickly now

Reply:
fair :)